src/main/java/org/openrewrite/kotlin/internal/KotlinPrinter.java

```diff
@@ -432,6 +432,19 @@ public J visitUnary(K.Unary unary, PrintOutputCapture<P> p) {
         return unary;
     }
 
+    @Override
+    public J visitUseSite(K.UseSite useSite, PrintOutputCapture<P> p) {
+        beforeSyntax(useSite, Space.Location.ANNOTATIONS, p);
```

Comment:
Not quite sure, but I suspect it should be like this:
```suggestion
        beforeSyntax(useSite, Space.Location.ANNOTATION_PREFIX, p);
```